oat-sa/extension-tao-itemqti
# Feature/tao 6386 sync min max choices

### PR Content 
1. Linting on the modules I plan to work on (you can filter mostly in the 3rd first commits)
 2. Creation of a dedicated component to handle min and max sync. See `/taoQtiItem/views/js/test/qtiCreator/widgets/component/minMax/test.html`
3. Need to add the `choiceDeleted` event, since the `deleted` event for a choice is unbound at the level of the interaction's widget.
4. Integration of the component in all interactions :  choice, order, associate, match, hottext, hotspot, graphicOrder, graphicAssociate, graphicGapMatch and selectPoint 

### How to test
 1. Run the unit test of the component
 2. Try to author the interactions mentioned above, play with min and max and check out the saved QTI.

Please note : 
 - removing a choice by clicking oustide the interaction leads to inconsistencies. This bug has been identified, and was already there. We will fix it in another task.
 - Changing the min/max value for the associate interaction doesn't update the displayed list of pairs. We will fix it in another task.
